tests/graph/test_subgraph_isomorphic.py

```diff
@@ -180,5 +180,27 @@ def test_non_induced_subgraph_isomorphic(self):
         for id_order in [False, True]:
             with self.subTest(id_order=id_order):
                 self.assertFalse(
-                    retworkx.is_subgraph_isomorphic(g_a, g_b, id_order=id_order)
+                    retworkx.is_subgraph_isomorphic(
+                        g_a, g_b, id_order=id_order, induced=True
+                    )
+                )
+            with self.subTest(id_order=id_order):
```

Comment:
If adding an additional subtest here I would add an additional parameter here to make it clear that this is for induced=False too. Something like:

```suggestion
            with self.subTest(id_order=id_order, induced=False):
```
(and adding `induced=True` to the above subTest)

That being said it's not critical though, because this really only controls the error message and not much else. It's not selectable for isolating execution or anything like that.

Reply:
Good point, added in f53404b